### PR TITLE
Makes antag AI law zero only show up to ghosts on antagHUD

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[tweak]

### CODE REVIEW

File: code/modules/mob/living/silicon/mommi/examine.dm [hunk 1]
```diff
@@ -46,10 +46,10 @@
 	to_chat(user, msg)
 
 	if(laws && isobserver(user) && !istype(user,/mob/dead/observer/deafmute)) //As a bastard child of robots, we don't call our parent's examine()
-		var/mob/dead/observer/fag = user
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/silicon/mommi/examine.dm:49]: Unatomic variable changes

Comment (SECBATON-GRIFFON) [reply code/modules/mob/living/silicon/mommi/examine.dm:49]: just the name, only used inside the proc

**Comment by WindowsErrors** [comment]
whats this supposed to be for again?

**Comment by Dacendeth** [comment]
Being able to read binary chat while dead is already a dead giveaway most of the time even if you couldn't inspect the laws.

**Comment by DamianX** [comment]
imagine removing stuff

**Pull request closed without merging**